commander/src/utils/core/database.ts

```diff
@@ -45,7 +45,9 @@ const isDbRunning = async (
 	installDir: string,
 	port: string,
 ): Promise<boolean> => {
-	const { stderr }: ExecResult = await exec(
+	const {
```

Comment:
I prefer the other way but if the chosen prettier rules are this format I'm ok

Reply:
before it was like this `const { stderr }: ExecResult = await exec(` I think with new prettier update, it might have changed. so it should be ok, as its gonna apply the same to other parts of the code.